lib/services/files-api-service.js

```diff
@@ -0,0 +1,107 @@
+// Copyright 2016, EMC, Inc.
+
+'use strict';
+
+var di = require('di');
+var fs = require('fs');
+var path = require('path');
+var express = require('express');
+var app = express();
+var walking = require('walk');
+
+module.exports = filesApiServiceFactory;
+
+di.annotate(filesApiServiceFactory, new di.Provide('Http.Services.Api.Files'));
+di.annotate(filesApiServiceFactory,
```

Comment:
I should have checked with others. I saw that the file structure used to implement it was named differently than other api calls so I made one which I saw as consistent. Would you advise that I move the staticfiles api call into "lib/services/file-service.js"?